battle.go

```diff
@@ -160,6 +164,10 @@ func (b *Battle) sortTurns(turns *[]TurnContext) {
 				if mvA.Priority() != mvB.Priority() {
 					return mvA.Priority() > mvB.Priority()
 				}
+				// Held item priority
+				if v, ok := pkmnA.metadata[MetaPriorityLast].(bool); ok && v {
+					return false
+				}
```

Comment:
Why can't you just check for the item being held?

Reply:
I suppose we can. I figured this would be a common enough case that other things can use the same meta tag to have it handled in the current/next round.

Reply:
I want to avoid using `metadata` as much as possible. It should only be used when no other option is possible.